src/components/text-input/TextInput.js

```diff
@@ -46,29 +45,28 @@ const TextInput = ({
     labelTooltip,
     ...rest
 }: Props) => {
-    const classes = classNames(className, 'text-input-container', {
-        'show-error': !!error,
+    const classes = classNames(className, 'bdl-TextInput-container', {
```

Comment:
We should ideally try to avoid qualifiers like `-container` at the root of components, since it indicates a descendent element. This should ideally just be `bdl-TextInput`.